health/micro-ui/web/console/Dockerfile

```
@@ -0,0 +1,30 @@
+FROM egovio/alpine-node-builder-14:yarn AS build
+#FROM ghcr.io/egovernments/alpine-node-builder-14:yarn AS build
+RUN apk update && apk upgrade
+RUN apk add --no-cache git>2.30.0
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)